### PR TITLE
[Import Wizard] update instructions

### PR DESCRIPTION
**Pull request by chejennifer**
- Add instructions about radio buttons
Screen Shot 2022-07-26 at 9 47 39 AM

### CODE REVIEW

**Review by beets — APPROVED** [review]

File: server/templates/import_wizard.html [hunk 1]
```diff
@@ -263,6 +263,8 @@
         <li>Confirm Mappings in the import wizard, one column at a time.
           <ul>
             <li>Identify Place, Variable, Date, and Observation Values Using the Tool</li>
+            <li>For each column title in the uploaded CSV file, the title will either be a specific [Place, Variable, Date, or Observation Value] (option 1 radio button) with corresponding values underneath it OR the column title will specify what the rows underneath it are: either a [Place, Variable, Date, or Observation Value] (option 2 radio button).
```
Comment (pradh) [inline server/templates/import_wizard.html:266]: Wonder if its visually easier to parse with bullet points?

e.g.
* Each column title in the uploaded CSV can specify:
   * a specific Place, Variable, Date or Observation Value - Option 1 radio button
   * or, that the rows underneath it are either a Place, Variable, Date or Observation Value - Option 2 radio button

File: server/templates/import_wizard.html [hunk 2]
```diff
@@ -263,6 +263,8 @@
         <li>Confirm Mappings in the import wizard, one column at a time.
           <ul>
             <li>Identify Place, Variable, Date, and Observation Values Using the Tool</li>
```
Comment (pradh) [inline server/templates/import_wizard.html:265]: nit: "... using the Wizard" or drop that

**Review by pradh — APPROVED** [review]
Thanks!

File: server/templates/import_wizard.html [hunk 3]
```diff
@@ -263,6 +263,8 @@
         <li>Confirm Mappings in the import wizard, one column at a time.
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply server/templates/import_wizard.html]: done

File: server/templates/import_wizard.html (hunk 1 above)
Comment (chejennifer) [reply server/templates/import_wizard.html]: done

File: server/templates/import_wizard.html (hunk 2 above)
Comment (chejennifer) [reply server/templates/import_wizard.html]: done

**Pull request merged**